scripts/20210421_has-allowed-email-domains/cleanup.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable */
+
+// Optional: Retrieve forms with fields that has a populated allowedEmailDomains but has no allowed email domains.
+db.getCollection('forms').aggregate([
```

Comment:
what does this query tell us?

Reply:
finding the inconsistent email fields itself so we have some forms to verify that the change has occurred